models/prices/prices_tokens.sql

```diff
@@ -514,6 +515,7 @@ VALUES
     ("orbs-orbs", "ethereum", "ORBS", "0xff56cc6b1e6ded347aa0b7676c85ab0b3d08b0fa", 18),
     ("osqth-opyn-squeeth", "ethereum", "oSQTH", "0xf1b99e3e573a1a9c5e6b2ce818b617f0e664e86b", 18),
     ("ost-ost", "ethereum", "OST", "0x2c4e8f2d746113d0696ce89b35f0d8bf88e0aeca", 18),
+    ("pal-paladin", "ethereum", "PAL", "0xab846fb6c81370327e784ae7cbb6d6a6af6ff4bf", 18),
```

Comment:
while this id is valid, it is inactive and prices won't come through: https://api.coinpaprika.com/v1/coins/pal-paladin